package.json

```diff
@@ -3,25 +3,25 @@
   "description": "The deepest card game under the sea",
   "version": "5.0.0",
   "dependencies": {
-    "lodash": "^4.17.21",
+    "chart.js": "^4.3.3",
+    "connect-redis": "3.0.2",
     "dayjs": "^1.11.9",
-    "sails": "1.5.3",
+    "lodash": "^4.17.21",
     "machinepack-passwords": "2.3.0",
+    "marked": "^5.1.1",
+    "sails": "1.5.3",
     "sails-disk": "^2.1.2",
     "sails-hook-orm": "2.1.1",
     "sails-hook-sockets": "1.5.5",
     "sails-postgresql": "4.0.0",
     "sails.io.js": "1.2.1",
-    "connect-redis": "3.0.2",
+    "socket.io-client": "2.3.0",
     "vue": "^3.3.4",
-    "vuex": "^4.1.0",
+    "vue-chartjs": "^5.2.0",
+    "vue-i18n": "^9.2.2",
     "vue-router": "^4.2.4",
     "vuetify": "^3.3.11",
-    "vue-i18n": "^9.2.2",
```

Comment:
These auto reorganized after installing husky, I'd avoid manually position them even if it's easier to reason about or read.